components/esp_lvgl_port/project_include.cmake

```diff
@@ -0,0 +1,65 @@
+# lvgl_port_create_c_image
```

Comment:
To me the usage of project_include.cmake looks fine since the purpose (I guess) is to make `lvgl_port_create_c_image` usage also from other component make makefiles, not only from esp_lvgl_port.

Reply:
> usage also from other component make makefiles, not only from esp_lvgl_port.

Hm, I could you the function from my `main` component this way. I'm not really sure how the function definition made its way to main/CMakeLists.txt, though.

OK, let's keep it as it is